src/main/java/seedu/masslinkers/logic/parser/ParserUtil.java

```diff
@@ -80,6 +96,7 @@ public static Phone parsePhone(String phone) throws ParseException {
     public static Telegram parseTelegram(String handle) throws ParseException {
         requireNonNull(handle);
         String trimmedHandle = handle.trim();
+        checkForPrefix(trimmedHandle);
```

Comment:
`add n/dxsds t/xsdsccdc dsxd` gives 
```
Telegram handle is invalid!
• Telegram handle must be at least 5 characters long.
• Telegram handle may only contain alphanumeric characters or underscores.
```
if we define telegram to be no space, will users say they are adding an extra parameter so the error should be ```Invalid arguments 'dsxd' instead?`

Reply:
Could add a note into UG that 
Any words that come after a prefix will be taken as the parameter. 
Eg.
`edit 1 n/john smith t/john parallelisation`
`john parallelization` would be considered as the parameter for telegram handle which is invalid. 
Ie. a space does not demarcate the end of a parameter

Reply:
Okay I've added it into the UG